core/src/main/scala/cats/UnorderedFoldable.scala

```diff
@@ -18,7 +18,7 @@ import cats.instances.long._
    * Returns true if there are no elements. Otherwise false.
    */
   def isEmpty[A](fa: F[A]): Boolean =
-    exists(fa)(Function.const(true))
+    !exists(fa)(Function.const(true))
 
```

Comment:
Tiny tiny nitpick but `nonEmpty` now negates `exists` twice. Maybe we should define `nonEmpty` as `exists(fa)(Function.const(true))` and then `isEmpty` as `!nonEmpty` WDYT? :)

Reply:
Actually I started from that :), but then I changed it as I had the impression it was reading better in my head (not exists instead of not non empty). But I don't really love my version either, as you said double negates, and even if it's simple logic, that makes reasoning convoluted and unnatural. I'm happy to change it 👍.